ui-components/src/navigation/SideNav.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from "react"
+import { NavigationContext } from "../config/NavigationContext"
+import "./SideNav.scss"
+
+export interface SideNavItemProps {
+  current?: boolean
+  url: string
+  label: string
+}
+
+export interface SideNavProps {
+  navItems?: SideNavItemProps[]
+}
+
+const SideNav = (props: SideNavProps) => {
+  const { LinkComponent } = React.useContext(NavigationContext)
+
+  return (
+    <nav className="side-nav" aria-label="Secondary navigation">
+      <ul>
+        {props.navItems?.map((navItem: SideNavItemProps, index: number) => {
+          if (navItem.current) {
+            return (
+              <li className="is-current" key={index}>
```

Comment:
For a11y, we should add `aria-selected` the current element

Reply:
Great catches! I'll get those styles updated and add that attribute. Thanks @emilyjablonski